packages/jellyfish-testing/src/index.ts

```diff
@@ -81,3 +84,50 @@ class TestingJsonRpcClient extends JsonRpcClient {
     })
   }
 }
+
+export class TestingGroup {
+  private constructor (
+    public readonly group: ContainerGroup,
+    public readonly testings: Testing[]
+  ) {
+  }
+
+  static create (n = 1): TestingGroup {
+    const containers: MasterNodeRegTestContainer[] = []
+    const testings: Testing[] = []
+    for (let i = 0; i < n; i += 1) {
+      const container = new MasterNodeRegTestContainer(GenesisKeys[i])
+      containers.push(container)
+
+      const testing = Testing.create(container)
+      testings.push(testing)
+    }
+    const group = new ContainerGroup(containers)
+
+    return new TestingGroup(group, testings)
+  }
+
+  get (index: number): Testing {
+    return this.testings[index]
+  }
+
+  length (): number {
+    return this.testings.length
+  }
+
+  async start (): Promise<void> {
+    return await this.group.start()
+  }
+
+  async stop (): Promise<void> {
+    return await this.group.stop()
+  }
+
```

Comment:
```suggestion

async exec(runner: (testing: Testing) => Promise<void>): Promise<void> {
  for each testing in this.testings {
      runner(testing)
  }
}

```

allowing you to do this

```ts
testingGroup.exec((testing: Testing) => {
  testing.rpc.misc....
})
```

and it will automatically run on all container